shared/components/NextCommunityEvent/index.js

```diff
@@ -94,30 +111,23 @@ class NextCommunityEvent extends React.Component {
                     {eventDateAndTime(startDateTime, endDateTime)}
                   </ExternalLink>
                 </li>
-                <li>
-                  <ExternalLink
-                    className="NextCommunityEvent__link--place"
-                    href={googleMapsUrl(location)}
-                    target="_blank"
-                    rel="noopener"
-                  >
-                    {eventLocation(location)}
-                  </ExternalLink>
-                </li>
-                {featuredEventDescription &&
+                <li>{this.renderLocationLink(location)}</li>
+                {featuredEventDescription && (
                   <li>
                     <p className="NextCommunityEvent__featured-description">
                       {featuredEventDescription}
                     </p>
                   </li>
-                }
+                )}
               </ul>
             </div>
             <TicketStatus {...statusProps} />
           </article>
         </div>
         <Talks talks={talks} collapsed={this.state.collapsed} />
-        {this.state.collapsed && <ReadMoreButton onClick={this.expandInfo.bind(this)} />}
+        {this.state.collapsed && (
+          <ReadMoreButton onClick={this.expandInfo.bind(this)} />
```

Comment:
Inline binding looks a bit messy, ideally if this codebase has good transpiling I'd have `expandInfo` as an arrow function so you wouldn't have to bind it here. Otherwise you can always bind stuff in the constructor.